lib/PayPal/Core/PayPalHttpConfig.php

```diff
@@ -68,6 +68,11 @@ public function __construct($url = null, $method = self::HTTP_POST, $configs = a
             //Remove the Cipher List for NSS
             $this->removeCurlOption(CURLOPT_SSL_CIPHER_LIST);
         }
+        if( array_key_exists('http.UseProxy',$configs) && $configs['http.UseProxy'] ){
+			if( !array_key_exists('http.Proxy',$configs) )
```

Comment:
Can we just make it such that if the `http.Proxy` exists, it means, we want to use proxy. Having additional `http.UseProxy` seems meaningless. Thoughts ?